yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -427,14 +446,35 @@ func (yb *TargetYugabyteDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, n
 			return false, nil
 		}
 
+		rowsStmt = fmt.Sprintf(
+			"SELECT count(*) FROM %s where migration_uuid='%s'", EVENTS_PER_TABLE_METADATA_TABLE_NAME, migrationUUID)
+		err = conn.QueryRow(context.Background(), rowsStmt).Scan(&rowCount)
+		if err != nil {
+			return false, fmt.Errorf("error executing stmt - %v: %w", rowsStmt, err)
+		}
+		if rowCount > 0 {
+			log.Info("table wise event meta info already created. Skipping init.")
+			return false, nil
+		}
+
 		for c := 0; c < numChans; c++ {
-			insertStmt := fmt.Sprintf("INSERT INTO %s VALUES ('%s', %d, -1)", EVENT_CHANNELS_METADATA_TABLE_NAME, migrationUUID, c)
+			insertStmt := fmt.Sprintf("INSERT INTO %s VALUES ('%s', %d, -1, %d, %d, %d)", EVENT_CHANNELS_METADATA_TABLE_NAME, migrationUUID, c, 0, 0 ,0)
 			_, err := conn.Exec(context.Background(), insertStmt)
 			if err != nil {
 				return false, fmt.Errorf("error executing stmt - %v: %w", insertStmt, err)
 			}
 			log.Infof("created channels meta info: %s;", insertStmt)
 		}
+
+		for _, tableName := range tableNames {
+			insertStmt := fmt.Sprintf("INSERT INTO %s VALUES ('%s', '%s', '%s', %d, %d, %d, %d)", EVENTS_PER_TABLE_METADATA_TABLE_NAME, migrationUUID, yb.tconf.Schema, tableName, 0, 0, 0, 0)
```

Comment:
BUG: `yb.tconf.Schema`!!
Use the `yb.tconf.Schema` only if the `tableName` is not qualified. Else, use the schema name present in the tableName.
Alternatively, if the `tableName` is already qualified, let's just drop the `schema_name` from the table. Directly write the qualified `tableName` in the stats table.